<launch>
  <node output="screen" name="rosapi" pkg="rosapi" type="rosapi.py" />
  <node output="screen" name="rosbridge" pkg="rosbridge_server" type="rosbridge.py" />
  <node output="screen" name="basic_controls" pkg="interactive_marker_tutorials" type="basic_controls" />
  <node output="screen" name="rviz" pkg="rviz" type="rviz" args="-d $(find demo_interactive_markers)/config/rviz_config.yaml" />
  
  <node name="im_tunnel" pkg="demo_interactive_markers" type="im_tunnel" />
  
</launch>
